src/plugins/intel_gna/src/backend/pwl_segments_creator_identity.cpp

```diff
@@ -114,7 +114,8 @@ void PWLSegmentsCreatorIdentity::UpdateSegmentOnTheLeftOf0_0(const gna_pwl_segme
     auto segment_1_in_int64 = ConvertSegmentTo64(segment_1);
 
     if (segment_1_in_int64.slope == 0) {
-        THROW_GNA_EXCEPTION << "Slope is 0 possible division by 0 when updating left segment!.";
+        THROW_GNA_EXCEPTION << "Slope is 0 possible division by 0 when updating left segment!. Check if your input "
+                               "data for POT were valid.";
```

Comment:
```suggestion
        THROW_GNA_EXCEPTION << "PWL segment slope is 0 - possible division by 0 when updating left segment!. Check if your input "
                               "data for POT are valid.";
```

Reply:
I suggest to add some more information about the particular activation function or other layer which caused the issue, if possible.